4.8/architectures

```diff
@@ -1,5 +1,5 @@
 bashbrew-arch   variants
-amd64     default,alpine,onbuild,slim,stretch,wheezy
```

Comment:
this should remain

Reply:
Should it? The Dockerfile is based on `alpine:3.4`, which doesn't have multi-arch support.

Reply:
amd64 exists, just not others

Reply:
Dur... brain's not firing on all cylinders! Haha... fix coming...